extract-sublime.sh

```diff
@@ -34,7 +34,7 @@ for file in *; do
   if test -f "$file"; then
     # Get its size
     #  `du $file | cut --fields 1`: `4172 plugin_host` -> `4172` (kb)
-    filesize="$(du "$file" | cut --fields 1)"
+    filesize="$(du "$file" | gcut --fields 1)"
```

Comment:
Could you generalize this? I think `gcut` is a macOS-ism. I normally run this script on Linux Mint, which only knows `cut`

Example at the top:
```bash
if $(which cut &> /dev/null); then
  CUT="cut"
elif $(which gcut &> /dev/null); then
  CUT="gcut"
else
  echo "Missing \`cut\` utility. Please install \`cut\` or \`gcut\`" 1>&2
  exit 1
fi

# ...

filesize="$(du "$file" | "$CUT" --fields 1)"
```

Reply:
Sorry that I forgot to revert my change to the script. I will do a force push.

Reply:
I think you have to reverse the order. BSD's `cut` does not understand the option `--fields`. That was exactly why I installed GNU's `gcut`.